test/aws.storage.test.js

```diff
@@ -21,145 +21,156 @@ redisStub.createClient = function() {
+  });
 });
 
 describe('Testing Get using aws', function() {
-
   it('Should not error out when the file exists', function() {
-    let spy = sinon.spy();
+    const spy = sinon.spy();
     s3Stub.getObject.returns({
       createReadStream: spy
     });
 
     storage.get('123');
     assert(spy.calledOnce);
-  })
+  });
 
   it('Should error when the file does not exist', function() {
-    let err = function() { throw new Error(); }
-    let spy = sinon.spy(err);
+    const err = function() {
+      throw new Error();
+    };
+    const spy = sinon.spy(err);
     s3Stub.getObject.returns({
       createReadStream: spy
     });
 
     assert.equal(storage.get('123'), null);
     assert(spy.threw());
-  })
+  });
 });
 
 describe('Testing Set using aws', function() {
   beforeEach(function() {
     expire.reset();
-  })
+  });
 
   after(function() {
     crypto.randomBytes.restore();
-  })
+  });
 
   it('Should pass when the file is successfully uploaded and no bitly key', function() {
     conf.bitly_key = null;
-    const buf =  new Buffer(10);
+    const buf = Buffer.alloc(10);
     sinon.stub(crypto, 'randomBytes').returns(buf);
     s3Stub.upload.callsArgWith(1, null, {});
-    return storage.set('123', {}, 'Filename.moz', 'url.com')
-                  .then(reply => {
-                      assert.equal(reply.uuid, buf.toString('hex'));
-                      assert.equal(reply.url, 'url.com');
-                      assert.notEqual(reply.uuid, null);
-                      assert(expire.calledOnce);
-                      assert(expire.calledWith('123', 86400000));
-                    })
-                  .catch(err => assert.fail());
-  })
+    return storage
+      .set('123', {}, 'Filename.moz', 'url.com')
+      .then(reply => {
+        assert.equal(reply.uuid, buf.toString('hex'));
+        assert.equal(reply.url, 'url.com');
+        assert.notEqual(reply.uuid, null);
+        assert(expire.calledOnce);
+        assert(expire.calledWith('123', 86400000));
+      })
+      .catch(err => assert.fail());
+  });
 
   it('Should fail if there was an error during uploading', function() {
     s3Stub.upload.callsArgWith(1, new Error(), null);
-    return storage.set('123', {}, 'Filename.moz', 'url.com')
-                  .then(reply => assert.fail())
-                  .catch(err => assert(1));
-  })
+    return storage
+      .set('123', {}, 'Filename.moz', 'url.com')
+      .then(_reply => assert.fail())
+      .catch(err => assert(1));
+  });
 });
 
 describe('Testing Delete from aws', function() {
   it('Returns successfully if the id is deleted off aws', function() {
     hget.callsArgWith(2, null, 'delete_token');
     s3Stub.deleteObject.callsArgWith(1, null, {});
-    return storage.delete('file_id', 'delete_token')
-                  .then(reply => assert(1), err => assert.fail())
-  })
+    return storage
+      .delete('file_id', 'delete_token')
+      .then(_reply => assert(1), err => assert.fail());
```

Comment:
Wasn't sure how we felt about the `.then(yay, boo)` syntax versus converting it to `.then(yay).catch(boo)` style, so I left these as-is.